CONTRIBUTORS.md

```diff
@@ -132,3 +132,4 @@
  - [jomp16](https://github.com/jomp16)
  - [Leon de Klerk](https://github.com/leondeklerk)
  - [CrispyBaguette](https://github.com/CrispyBaguette)
+ - [v0idMrK](https://github.com/v0idMrK)
```

Comment:
Should be in another list (Jellyfin contributors).

Reply:
Oh maybe I misread on the contribution guide.

Will check later today thank you :)